utils/data.rb

```diff
@@ -0,0 +1,16 @@
+require 'yaml'
+
```

Comment:
Please redo this class methods 
http://rubygarage.github.io/slides/ruby-basics#/11